### ISSUE TITLE
Permission denied: Disable "phar.readonly" and "phar.require_hash"

### ISSUE THREAD

**Issue by ghost** [mpyw/comphar#3]
I am trying to pack [diff-parser](https://github.com/ptlis/diff-parser)
I kept getting `Permission denied: Disable "phar.readonly" and "phar.require_hash"`

**Comment by shino47** [comment]
Change `;phar.readonly = On` to `phar.readonly = Off` in your `php.ini`. Notice the semicolon. Usually you can find this file in `/etc/php/7.x/cli/php.ini`

**Comment by mpyw** [comment]
The problem is php.ini related, not a library bug. Please feel free to submit the PR that updates documentation! Thanks

**Comment by BurakBoz** [comment]
Nope! I've tried editing php.ini this is not effective! same error again and again nothing is resolved with php.ini changes.
on windows run command:
`where comphar`
> C:\Users\asus\AppData\Roaming\Composer\vendor\bin\comphar
C:\Users\asus\AppData\Roaming\Composer\vendor\bin\comphar.bat

edit comphar.bat find
`php "%BIN_TARGET%" %*`
replace
`php -d phar.readonly=off -d phar.require_hash=off  "%BIN_TARGET%" %*`

if you are on linux platform **`which comphar`** command will do the same job, simply add 2 parameters after php command 
`php -d phar.readonly=off -d phar.require_hash=off `

Good luck ,)

**Comment by lordspace** [comment]
this should only be enabled on dev/staging servers for security reasons
